src/lib/chat/matrix-client.ts

```diff
@@ -238,12 +248,36 @@ export class MatrixClient implements IChatClient {
     }
 
     const result = await this.matrix.createRoom(options);
+
     // Any room is only set as a DM based on a single user. We'll use the first one.
     await setAsDM(this.matrix, result.room_id, users[0].matrixId);
 
+    await this.sendAdminMessage(result.room_id, users[0].userId, currentUserId, adminMessageType);
```

Comment:
I don't really think this should be tied up in the matrix-client. Sending this welcome message is business logic and should be handled in the sagas